spec/model/dirty_spec.rb

```diff
@@ -8,6 +8,7 @@
         builder.use Her::Middleware::FirstLevelParseJSON
         builder.use Faraday::Request::UrlEncoded
         builder.adapter :test do |stub|
+          stub.get("/users") { |env| [200, {}, [{ :id => 1, :fullname => "Lindsay Fünke" }].to_json] }
```

Comment:
Can we add multiple users here for [consistency](https://github.com/remiprev/her/blob/master/spec/model/orm_spec.rb#L13)?